### DISCUSSION TITLE
ADD plugins buy land or rent in land or sub land

### DISCUSSION THREAD

**Discussion by UnknowUser0** [GriefPrevention/GriefPrevention#2230] [Ideas]
I thought it might be a good thing to add this plugin to boost the economy in the game

**Comment by bigpresh** [comment]
There's already economy integrations e.g. Vault etc to allow buying/selling claim _blocks_; not sure about selling / "renting" claims themselves - https://docs.griefprevention.com/configuration/#economy-integration

Any further extensions to that kind of thing wouldn't be something that belongs in GriefPrevention's core, IMO, but could easily be done as addons.
